chat/src/utils/events.ts

```diff
@@ -0,0 +1,187 @@
+import { snakeCase } from "lodash";
```

Comment:
1980

Reply:
changed it
https://www.npmjs.com/package/snakecase-keys